config/routes.rb

```diff
@@ -190,6 +190,11 @@
   end
 
   resources :school_moves, path: "school-moves", only: %i[index show update]
+  resources :school_move_exports,
+            path: "school-move-exports",
+            only: %i[create show update] do
+    get "download", on: :member
+  end
```

Comment:
I would suggest putting this inside the school move resource to keep the paths tidy. Something like:

```rb
resources :school_moves, path: "school-moves", only: %i[index show update] do
  resources :exports, only: %i[create show update] do
    get "download", on: :member
  end
end
```

We can rename the controller to `SchoolMoves::ExportController`.